knowledge_base/Database-Monitoring.md

```diff
@@ -1,30 +1,39 @@
-# PSQL Schema Inspection
+# Database Monitoring
 
-## How to Connect
+_See also the [Datadog](./Datadog.md) and [Heroku](./Heroku.md) entries._
+
+## Contents
+
+- [Heroku Database Monitoring](#heroku-database-monitoring)
+- [Schema Inspection](#schema-inspection)
+- [Local Database Monitoring](#local-database-monitoring)
+  * [PGAdmin4](#pgadmin4)
+  * [Helper Queries](#helper-queries)
+- [Change Log](#change-log)
+
+## Heroku Database Monitoring
 
 To connect to the PSQL schema, use the following command:
 
 ```bash
 heroku pg:psql postgresql-clear-46785 -a commonwealth-staging
 ```
 
-## About the Database
+As of 230413, the current database being used is Postgres 13.10. You can retrieve general information and uptime using the following SQL queries:
```

Comment:
We are on version 15.7 now. Major version (15) will not change without a manual update but the minor version may be updated by Heroku.